Integrations/integration-FidelisElevateNetwork.yml

```diff
@@ -0,0 +1,777 @@
+description: Automate Detection and Response to Network Threats and data leakage in
+  your organization with Fidelis Elevate Network Integration.
+configuration:
+- display: Server URL
+  name: server_url
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Trust any certificate (unsecure)
+  name: unsecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+- display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3     months,
+    1 year)
+  name: fetch_time
+  defaultvalue: 3 days
+  type: 0
+  required: false
+script:
+  script: |-
+    ''' IMPORTS '''
+    import json
+    import shutil
+    import requests
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+
+    ''' GLOBALS / PARAMS '''
+    IS_FETCH = demisto.params()['isFetch']
+    INCIDENT_TYPE = demisto.params()['incidentType']
+    SERVER_URL = demisto.params()['server_url']
+    CREDENTIALS = demisto.params()['credentials']
+    INSECURE = demisto.params()['unsecure']
+    PROXY = demisto.params()['proxy']
+    FETCH_TIME = demisto.params().get('fetch_time', '3 days')
+    SESSION_ID = None
+
+    if not demisto.params().get('proxy', False):
+        del os.environ['HTTP_PROXY']
+        del os.environ['HTTPS_PROXY']
+        del os.environ['http_proxy']
+        del os.environ['https_proxy']
+
+
+    ''' HELPER FUNCTIONS '''
+    def http_request(method, url_suffix, params=None, data=None, files=None, is_json=True):
+        # A wrapper for requests lib to send our requests and handle requests and responses better
+        headers = {}
+        if SESSION_ID is not None:
+            headers['x-uid'] = SESSION_ID
+            if files is None:
+                headers['Content-Type'] = 'application/json'
+
+        res = requests.request(
+            method,
+            SERVER_URL + url_suffix,
+            data=None if data is None else json.dumps(data),
+            headers=headers,
+            params=params,
+            files=files,
+            verify=not INSECURE,
+        )
+
+        # Handle error responses gracefully
+        if res.status_code not in {200, 201}:
+            if res.status_code == 500:
+                error = res.json()
+                return_error('Error in API call to Fidelis Integration [%d] - %s' % (res.status_code, error['detailMessage']))
+            else:
+                return_error('Error in API call to Fidelis Integration [%d] - %s' % (res.status_code, res.reason))
+
+        if is_json:
+            return res.json()
```

Comment:
catch the error of the json parse.  If failed to parse json then print the error with the original res.content